web/src/compose_validate.ts

```diff
@@ -409,6 +409,21 @@ export function warn_if_topic_resolved(topic_changed: boolean): void {
     // Pass topic_changed=true if this function was called in response
     // to a topic being edited.
 
+    if (
+        compose_state.get_is_processing_forward_message() &&
+        !compose_state.get_is_stream_recipient_dropdown_closed()
+    ) {
+        // This is for the case of forwarding a message when the
+        // channel picker is opened. There is a possibility that
+        // this banner might be displayed at the same time the
+        // channel picker is opened. We don't want that situation.
+        // Therefore, we are preventing the display of this
+        // banner when the channel picker is opened.
+        return;
+    }
+
+    compose_state.set_is_stream_recipient_dropdown_closed(false);
```

Comment:
why is this set to false here?

Reply:
A better place to set it would be `onShow ` callback of the dropdown so that other libraries can use it as well if required.

Reply:
https://github.com/zulip/zulip/pull/33903#issuecomment-2850390457